apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -1254,31 +1242,29 @@ class LivechatClass {
 		const scopeData = scope || (nextDepartment ? 'department' : 'agent');
 		this.logger.info(`Storing new chat transfer of ${room._id} [Transfered by: ${_id} to ${scopeData}]`);
 
-		await sendMessage(
-			transferredBy,
-			{
-				t: 'livechat_transfer_history',
-				rid: room._id,
+		const transferMessage = {
+			t: 'livechat_transfer_history',
+			rid: room._id,
+			ts: new Date(),
+			msg: '',
+			u: {
+				_id,
+				username,
+			},
+			...(transferData.transferredBy.userType === 'visitor' && { token: room.v.token }),
+			groupable: false,
```

Comment:
all these properties are already set inside of `saveSystemMessageAndNotifyUser` so we don't need to pass them
```suggestion
			...(transferData.transferredBy.userType === 'visitor' && { token: room.v.token }),
```